package.json

```diff
@@ -10,6 +10,7 @@
   "main": "app/index.js",
   "scripts": {
     "migrate": "sequelize db:migrate",
+    "import": "scripts/import.js example-data/testfile.csv",
```

Comment:
There's no other reference to this file name. Are we assuming that the user will redirect the output of the new `maketest.js` into `example-data/testfile.csv`?